BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IMachineService.java

```diff
@@ -0,0 +1,65 @@
+package com.brewmes.common.services;
+
+import com.brewmes.common.entities.Connection;
+import com.brewmes.common.util.Command;
+import com.brewmes.common.util.Products;
+
+import java.util.List;
+
+public interface IMachineService {
+
+    /**
+     * Takes a command and sends it to the desired machine
+     * @param command The command that should be sent to the machine
+     * @param machineId The ID of the machine that should receive the command
+     */
+    void controlMachine(Command command, String machineId);
```

Comment:
```suggestion
    boolean controlMachine(Command command, String machineId);
```
Should probably return a response if the command was successful or not.